docs/ossec_alerts.rst

```diff
@@ -527,32 +565,41 @@ securedrop@freedom.press for help.
 
 .. _SecureDrop Support Portal: https://securedrop-support.readthedocs.io/en/latest/
 
+Daily reports
+^^^^^^^^^^^^^
+
+On days where file integrity checksums have changed or users have logged into ``app``
+or ``mon`` servers, you will receive emails entitled ``Daily report: File changes`` or
+``Daily report: Successful logins``. These emails may be a more digestible format
+should you not have continuous access to the inbox or GPG key.
+
+**Action**: periodically review these daily reports to ensure file changes correspond
+to platform updates and logins correspond to authorized admin activity on the SecureDrop
+servers.
+
+If you have any suggestions on how to further tune or improve the alerting,
+you can join the discussion or open an issue on `GitHub <https://github.com/freedomofpress/securedrop/labels/goals%3A%20reduce%20IDS%20noise>`__.
+
 Uncommon OSSEC Alerts
 ~~~~~~~~~~~~~~~~~~~~~
 
-.. _uncommon_alerts:
+Logins
+^^^^^^
+Ossec will send an alert whenever a user interactively logs into the system (via SSH
+or console). You will receive an email containing the following. ::
 
-SecureDrop also runs automatic checks for submission data integrity
-problems. For example, secure deletion of large submissions could
-potentially be interrupted, resulting in an alert recommending steps
-to :ref:`clean them up <submission-cleanup>`.
+    Rule: 40101 (level 12) -> 'System user successfully logged to the system.'
```

Comment:
On my prod install, I see a level 3 alert `Rule: 5501 (level 3) -> 'Login session opened'` on interactive logins. I don't see a level 12 alert in ~2 years of runtime. Note though that I only see these 5501 alerts in the logfile on the mon server, not in my inbox; other alerts (for both app and mon) have been happily sent throughout this time period.

Reply:
I think those used to happen on first ssh login, I also went through my logs and you are correct, I don't see these anymore